ui/app/models/job.js

```diff
@@ -28,6 +28,7 @@ export default class Job extends Model {
   @attr('number') createIndex;
   @attr('number') modifyIndex;
   @attr('date') submitTime;
+  @attr('string') nodePool; // Jobs are related to Node Pools via Allocations, but no relationship.
```

Comment:
```suggestion
  @attr('string') nodePool; // Jobs are related to Node Pools either directly or via its Namespace, but no relationship.
```

The job can be directly assigned to a node pool in the jobspec or inherit the default pool from the namespace. 

I'm not sure what `but no relationship` means in this context, is it something like `@belongsTo('node-pool')`?